### DISCUSSION TITLE
Remplacer l'évier magique par un lavoir

### DISCUSSION THREAD

**Discussion by lemaximedu66** [Crownicles/Crownicles#1317] [Suggestions fermées]
Il le dit à chaque fois qu'on jette une potion ce serait bien de le remplacer par un lavoir

**Comment by Miala-python** [comment]
Oui car il n'y a pas d'évier n'importe où lol

**Reply by Pietagorh** [reply]
Bien sûr que si
Les routes du royaume sont très bien canalisées, alors un mage a décidé de faire pop des éviers un peu partout
![unknown-46.png](https://user-images.githubusercontent.com/60353007/182036364-79c2a7b5-d922-4be0-97c0-31f98d657b18.png)

**Reply by lemaximedu66** [reply]
Mec t un génie

**Reply by Pietagorh** [reply]
Je sais

**Reply by lemaximedu66** [reply]
Mais je reste quand même sur mon avis de départ 😉

**Reply by lemaximedu66** [reply]
Tu l'as vraiment fait juste pour moi

**Reply by BastLast** [reply]
Non c'est un vieux truc qui a été posté dans le salon même il y a très longtemps

**Comment by Feiryn** [comment]
Un lavoir ça servait à nettoyer le linge, pas à jeter ses déchets

**Reply by lemaximedu66** [reply]
On s'en fout lol un évier ça sert pas à ça non plus

**Reply by QQtin3** [reply]
Alors il me semble quand même qu'un évier permet justement de vider des liquides, l'évier te permet bien de vider l'eau que tu n'utilises pas du robinet n'est-ce pas ?

**Reply by Feiryn** [reply]
Oui c'est l'intérêt 1er d'un évier, jeter les liquides usagés

**Comment by lemaximedu66** [comment]
Oe mais genre tu fais pas Spawn des éviers comme ça c bizarre. es ce que tu
sors de chez toi et tu trouve un évier en plein millieu de la nature ????

**Reply by BastLast** [reply]
Bon je met un terme a cette conv, ca spam mes mails c'est relou
